src/modules/pages/components/Dashboard/Admin/AddPayment/AddPayment.jsx

```diff
@@ -110,7 +110,9 @@ const AddPayment = ({ colonyClient, network, user, wallet }: Props) => {
       // eslint-disable-next-line no-undef
       fetch(
         // eslint-disable-next-line max-len
-        `${server}/api/contribution?sessionID=${user.session.id}&address=${wallet.address}&network=${network.slug}`,
+        `${server}/api/contribution?sessionID=${user.session.id}&address=${
```

Comment:
If we change to this, we no longer need `eslint-disable-next-line`. My editor still prefers that former. :thinking:

Reply:
Hm, not sure how this ended up in here - I've updated this to match what `yarn lint` yields